embark-skills.lua

```diff
@@ -42,13 +42,13 @@ function adjust(dwarves, callback)
 end
 
 local scr = dfhack.gui.getCurViewscreen() --as:df.viewscreen_setupdwarfgamest
-if dfhack.gui.getCurFocus() ~= 'setupdwarfgame' or scr.show_play_now == 1 then
+if not df.viewscreen_setupdwarfgamest:is_instance(scr) then
```

Comment:
```suggestion
if not dfhack.gui.matchFocusString('setupdwarfgame/Dwarves', scr) then
```

Reply:
matchFocusString makes sure you're also on the correct tab

Reply:
In this case should the error message be updated to not only state must be on the "Prepare carefully" screen but also the Dwarves tab?

Reply:
that sounds like a good idea -- maybe also add that to the docs if it's not already there